packages/@react-aria/textfield/src/useTextField.ts

```diff
@@ -12,16 +12,20 @@
 
 import {AriaTextFieldProps} from '@react-types/textfield';
 import {ChangeEvent, InputHTMLAttributes, LabelHTMLAttributes, RefObject, TextareaHTMLAttributes} from 'react';
-import {ElementType} from 'react';
+import {ElementType, HTMLAttributes} from 'react';
 import {filterDOMProps, mergeProps} from '@react-aria/utils';
+import {useField} from '@react-aria/label';
 import {useFocusable} from '@react-aria/focus';
-import {useLabel} from '@react-aria/label';
 
 export interface TextFieldAria {
```

Comment:
Should @react-aria interfaces like TextFieldAria extend [FieldAria](https://github.com/adobe/react-spectrum/pull/1846/files#diff-012e9e517a76baacb63e3e4d93716f8d0acd4dadf5477073cae223ca3832e9d0R20)? That'll provide `labelProps`, but will also provide `fieldProps` which I'm not sure we want.

Reply:
I think what you have here should be fine. If you do want to extend `FieldAria`, you could always use `extends Omit<FieldAria, 'fieldProps'>`

Reply:
I do wonder if we could get away with not including `useField` in `useTextField` and instead having users call it separately, but this does seem like a common enough use case to include within `useTextField`

Reply:
Would it make more sense once `useField` also gets incorporated in other hooks like `useSelect`? I was mainly following this from the issue desc:

> Once we have this hook, it should replace the call to useLabel in the hooks for all components where help text is supported. These are: useTextField, useSelect, useComboBox, useRadioGroup, and useCheckboxGroup.

Reply:
Yeah I think what you have makes sense, was really just pondering to myself whether or not things like `description` and `error` props were common enough to include inside those hooks.